java/BUILD

```diff
@@ -0,0 +1,22 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+
+docker_build(
+    name = "java8",
+    base = "//base:base",
+    debs = [
+        "@zlib//file",
+        "@libfreetype//file",
+        "@libpng//file",
```

Comment:
I think libfreetype and libpng were me messing around trying to get Jenkins running.  I don't think they're needed for a properly minimal setup.